### ISSUE TITLE
Problem installing base v0.9.3( 4.06 support )

### ISSUE THREAD

**Issue by pe200012** [janestreet/base#28]
```
#=== ERROR while compiling base.v0.9.3 ========================================#
# context      2.0.0~beta4 | linux/x86_64 | ocaml-variants.4.06.0+trunk+fp+flambda | https://opam.ocaml.org/2.0#2017-10-29 02:04
# path         ~/.opam/4.06.0+trunk+fp+flambda/.opam-switch/build/base.v0.9.3
# command      jbuilder build -p base -j 3
# exit-code    1
# env-file     ~/.opam/log/base-9566-ffb3fd.env
# output-file  ~/.opam/log/base-9566-ffb3fd.out
### output ###
# Error: Signature mismatch:
# [...]
# (cd _build/default && /home/pe200012/.opam/4.06.0+trunk+fp+flambda/bin/ocamlc.opt -w -40 -g -bin-annot -I /home/pe200012/.opam/4.06.0+trunk+fp+flambda/lib/sexplib/0 -I compiler-stdlib/src -I shadow-stdlib/src -no-alias-deps -I src -open Base__ -o src/base__String.cmo -c -impl src/string.ml)
# File "src/string.ml", line 110, characters 5-140:
# Error: Signature mismatch:
#        ...
#        Values do not match:
#          val create : len:int -> bytes
#        is not included in
#          val create : len:int -> t
#        File "src/blit_intf.ml", line 77, characters 2-27:
#          Expected declaration
#        File "src/string.ml", line 112, characters 10-16: Actual declaration

```

What does it mean? How can I fix it?

**Comment by copy** [comment]
Also running into this. This is due to a backwards-incompatible change in OCaml 4.06. I'd suggest changing the title to "4.06 support".

**Comment by seliopou** [comment]
I believe this issue has been resolved in master, so a quick workaround would be to use a pin. The next release is coming shortly. We're working on getting all of your publicly-released packages to be `-safe-string`-compatible, and I believe that the majority of the work is done.

**Comment by hhugo** [comment]
base.v0.9.4 is now available in opam and is compatible with OCaml 4.06.
